Zend/Zend.m4

```diff
@@ -1,5 +1,22 @@
 dnl This file contains Zend specific autoconf macros.
 
+AC_DEFUN([ZEND_CHECK_TYPEDEF_REDECL], [dnl
```

Comment:
```suggestion
dnl
dnl ZEND_CHECK_TYPEDEF_REDECL
dnl
dnl Check whether the compiler supports typedef redeclaractions. For example,
dnl GCC 4.2 on Mac OS X 10.6 emits a redefinition of typedef error by default.
dnl
AC_DEFUN([ZEND_CHECK_TYPEDEF_REDECL], [dnl
```

Just some minor macro "header" to describe what the macro does so it's clearer in the future why this check. Or it can be adjusted according the main issue.

Reply:
Or even better to describe that this version has the -std=c89 set on by default.